.gitignore

```diff
@@ -52,7 +52,9 @@ target/
 logs/
 
 dependency-reduced-pom.xml
+checkstyle_config.xml
 
 Transactions2014.csv
 TestTransactions.csv
```

Comment:
I should have spotted this in the last PR, but should this file be checked in since the tests depend on it?